<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapWindow</class>
 <widget class="QMainWindow" name="MapWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="ToolsWidget" native="true">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>0</y>
      <width>450</width>
      <height>721</height>
     </rect>
    </property>
    <widget class="QLabel" name="start_title">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>370</y>
       <width>40</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>起点</string>
     </property>
    </widget>
    <widget class="QLabel" name="end_title">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>432</y>
       <width>40</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>终点</string>
     </property>
    </widget>
    <widget class="QPushButton" name="calc_button">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>350</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;幼圆&quot;;
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:1 rgba(12, 226, 10, 255));</string>
     </property>
     <property name="text">
      <string>计算</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reset_button">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>420</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;幼圆&quot;;
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:1 rgba(255, 21, 21, 255));</string>
     </property>
     <property name="text">
      <string>重置</string>
     </property>
    </widget>
    <widget class="QComboBox" name="start_point">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>370</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maxVisibleItems">
      <number>20</number>
     </property>
    </widget>
    <widget class="QComboBox" name="end_point">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>432</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maxVisibleItems">
      <number>20</number>
     </property>
    </widget>
    <widget class="QLabel" name="add_route">
     <property name="geometry">
      <rect>
       <x>164</x>
       <y>104</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>添加线路</string>
     </property>
    </widget>
    <widget class="QLabel" name="add_station">
     <property name="geometry">
      <rect>
       <x>164</x>
       <y>5</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>添加站点</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_station_button">
     <property name="geometry">
      <rect>
       <x>334</x>
       <y>45</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;幼圆&quot;;
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:1 rgba(21, 217, 255, 255));</string>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QLabel" name="add_route_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>310</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>出行规划</string>
     </property>
    </widget>
    <widget class="QComboBox" name="to_station">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>214</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maxVisibleItems">
      <number>20</number>
     </property>
    </widget>
    <widget class="QLabel" name="u_station">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>154</y>
       <width>60</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>站点一</string>
     </property>
    </widget>
    <widget class="QLabel" name="v_station">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>214</y>
       <width>60</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>站点二</string>
     </property>
    </widget>
    <widget class="QComboBox" name="from_station">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>154</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maxVisibleItems">
      <number>20</number>
     </property>
    </widget>
    <widget class="QPushButton" name="add_path_button">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>172</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;幼圆&quot;;

background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:1 rgba(243, 255, 21, 255));</string>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="station_input">
     <property name="geometry">
      <rect>
       <x>124</x>
       <y>50</y>
       <width>191</width>
       <height>33</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="station_name">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>52</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>站点名称</string>
     </property>
    </widget>
    <widget class="QLabel" name="exchange_title">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>595</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>换乘信息</string>
     </property>
    </widget>
    <widget class="QLabel" name="Line_Title">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>260</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>所属线路</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Line_input">
     <property name="geometry">
      <rect>
       <x>227</x>
       <y>260</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 63 11pt &quot;A-OTF Shin Maru Go Pro DB&quot;;</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
    </widget>
    <widget class="QTextBrowser" name="route_info">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>510</y>
       <width>301</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="GraphView" name="MapWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>830</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphView</class>
   <extends>QWidget</extends>
   <header>graphview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
